src/DataFactory/DataFactoryV2/help/Set-AzDataFactoryV2IntegrationRuntime.md

```diff
@@ -173,6 +176,51 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -DataProxyIntegrationRuntimeName
+The Self-Hosted Integration Runtime name which is used as a proxy
+
+```yaml
+Type: System.String
+Parameter Sets: ByIntegrationRuntimeName, ByResourceId, ByIntegrationRuntimeObject
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DataProxyStagingLinkedServiceName
+The Azure Blob Storage Linked Service name that references the staging data stroe to be used when moving data between Self-Hosted and Azure-SSIS Integration Runtime
+
+```yaml
+Type: System.String
+Parameter Sets: ByIntegrationRuntimeName, ByResourceId, ByIntegrationRuntimeObject
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -DataProxyStagingPath
```

Comment:
Can you add an example using the new parameters?

Reply:
Added. Thanks